### ISSUE TITLE
Latest GIMP beta has trouble with G'Mic

### ISSUE THREAD

**Issue by FakeShemp** [flathub/org.gimp.GIMP.Plugin.GMic#39]
The latest GIMP beta throws some warnings when starting and fails to load G'Mic.

```
/app/extensions/plug-ins/GMic/GMic: symbol lookup error: /app/extensions/plug-ins/GMic/GMic: undefined symbol: gimp_image_set_active_layer
GIMP-WARNING: gimp-2.99: gimp_wire_read(): error

/app/lib/gimp/2.99/plug-ins/file-aa/file-aa: error while loading shared libraries: libslang.so.2: cannot open shared object file: No such file or directory
GIMP-WARNING: gimp-2.99: gimp_wire_read(): error
```

**Comment by hfiguiere** [comment]
looks like the API/ABI changed but not on GMIC side.

**Comment by hfiguiere** [comment]
https://github.com/GreycLab/gmic/issues/411

**Comment by hfiguiere** [comment]
See also
https://github.com/c-koi/gmic-qt/issues/163

**Comment by hfiguiere** [comment]
3.2.0 doesn't fix this.

**Comment by hfiguiere** [comment]
Closed wtih #53
